impl/client/src/main/javascript/web/pentaho/ccc/visual/barHorizontal.js

```diff
@@ -14,27 +14,27 @@
  * limitations under the License.
  */
 define([
-  "module",
-  "pentaho/visual/models/barHorizontal",
-  "./barAbstract"
-], function(module, modelFactory, baseViewFactory) {
+  "module"
+], function(module) {
 
   "use strict";
 
-  return function(context) {
+  return [
+    "pentaho/ccc/visual/barAbstract",
```

Comment:
Did we lost the ability of using relative namespaces?

Reply:
In the initial development of the "module dependency id" concept yes. Later, I had to add that ability back, to support relative ids to sibling modules in OSGI projects, where we don't know the base module id.